### PR TITLE
Added file process listener

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Event/FileProcessListener.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     * @return void
+     */
+    public function preUpdate(File $file, LifecycleEventArgs $args): void
+    {
+        $file = $args->getObject();
+        $entityManager = $args->getEntityManager();
+        $unitOfWork = $entityManager->getUnitOfWork();
+        $metadata = $entityManager->getClassMetadata(get_class($file));
+        if ($file instanceof File) {
+            if ($file->getFileSha256Hash()) {
```
Comment (mickel1138) [inline src/Event/FileProcessListener.php:26]: We may also want to check if the status is "In Progress". Also should we check other fields for completeness?

Comment (praneethpr) [reply src/Event/FileProcessListener.php]: Fixed it

[the remark this answers is not in the document]
Comment (praneethpr) [reply src/Event/FileProcessListener.php]: Changed it to empty

**Review by mickel1138 — APPROVED** [review]
Works well!

**Pull request merged**